code/linear-regression/src/Components/ScrollyRegression.svelte

```diff
@@ -150,16 +150,16 @@
           <p>
             The goal of linear regression is reducing this error such that we
             find a line/surface that 'best' fits our data. For our simple
-            regression problem, that involves estimating the slope and intercept
-            of our model, {@html katexify(`\\hat{\\beta_1}`, false)} and {@html katexify(
-              `\\hat{\\beta_0}`,
+            regression problem, that involves estimating the y-intercept and slope
+            of our model, {@html katexify(`\\hat{\\beta_0}`, false)} and {@html katexify(
+              `\\hat{\\beta_1}`,
               false
             )}.
             <br /><br />
             For our specific problem, the best fit line is shown. There's still error,
-            sure, but the general pattern is captured well. As a result, we can be
-            reasonably confident that were we to plug in future values of square-footage,
-            our predicted values of price would be fairly accurate.
+            but the general pattern is captured well. As a result, we can be
```

Comment:
Let's keep the sure:

sure, but the general pattern is captured well. As a result, we can be reasonably confident that if we plug in new values of square-footage, our predicted values of price would be reasonably accurate.